benchmark/src/main/scala/geotrellis/pointcloud/raster/ept/IDWRasterSourceResolutionBench.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.pointcloud.raster.ept
+
+import geotrellis.raster.{CellSize, GridExtent, MultibandTile, Raster, RasterExtent}
+import geotrellis.raster.io.geotiff.Auto
+import geotrellis.proj4.LatLng
+
+import org.openjdk.jmh.annotations._
+import java.util.concurrent.TimeUnit
+
+@BenchmarkMode(Array(Mode.AverageTime))
+@State(Scope.Benchmark)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+class IDWRasterSourceResolutionBench {
+  val catalogPath = "../pointcloud/src/test/resources/red-rocks"
+
+  /** This bench benchmarks the actual PDAL reads, not the RasterSource initialization time. */
+  val rs   = IDWRasterSource(catalogPath)
+  val rsAuto2 = IDWRasterSource(catalogPath)
+
+  /**
+    * jmh:run -i 3 -wi 1 -f1 -t1 .*IDWRasterSourceResolutionBench.*
+    *
+    * jmh:run -i 10 -wi 5 -f1 -t1 .*IDWRasterSourceResolutionBench.*
+    *
+    */
+
+  @Benchmark
+  def IDWRasterSourceNaturalRead(): Option[Raster[MultibandTile]] = rs.read()
+
+  @Benchmark
+  def IDWRasterSourceAuto2Read(): Option[Raster[MultibandTile]] = rsAuto2.read()
```

Comment:
what is the difference between rs and rsAuto2 here?

Reply:
Sorry.  `rsAuto2` was supposed to use the `Auto(2)` overview strategy, but I could never get this benchmark to run without an OOM, so it never got exercised.

Reply:
@jpolchlo wow, that's concerning, `Auto(2)` is supposed to fetch a less resolute data, do you know why OOM had happened?

Reply:
The OOM happened with the default AutoHigherResolution.  Either the VM settings need to be tweaked or the AOI needs to be narrowed to make this test go.  Never got a chance to come back to that work.

Reply:
@jpolchlo OK, I will take car of it and will launch the benchmark before merging the PR.

Reply:
I see what is going on, I'll look how we can merge it in or can we at all merge it in with preserving the old behavior.